Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -15894,6 +15898,10 @@ INSTANTIATE_SEND_WITH_ASYNC_REPLY_TO_PROCESS_CONTAINING_FRAME(WebPage::NavigateS
 INSTANTIATE_SEND_WITH_ASYNC_REPLY_TO_PROCESS_CONTAINING_FRAME(WebPage::DrawToImage);
 INSTANTIATE_SEND_WITH_ASYNC_REPLY_TO_PROCESS_CONTAINING_FRAME(WebPage::DrawToPDFiOS);
 #endif
+#if PLATFORM(MAC)
```

Comment:
Same
```suggestion
#if ENABLE(PDF_HUD)
```

Reply:
Actually, ignore my suggestion, it looks like this has other uses besides the HUD.